integrations/salesforce/subscription.go

```diff
@@ -134,6 +136,16 @@ func (h handler) eventLoop(ctx context.Context, l *zap.Logger, clientID string,
 					continue
 				}
 
+				// Save the latest replay ID.
+				replayID := msg.LatestReplayId
+				encodedReplayID := base64.StdEncoding.EncodeToString(replayID)
```

Comment:
I don't see `replayID` being used anywhere else in this function except in line 140, so this makes more sense to me:

```suggestion
				replayID := base64.StdEncoding.EncodeToString(msg.LatestReplayId)
```